packages/synthetic-chain/src/cli/doctor.ts

```diff
@@ -0,0 +1,68 @@
+import fs from 'node:fs';
+import path from 'node:path';
+import { ProposalInfo } from './proposals.ts';
+import assert from 'node:assert';
+import { execSync } from 'node:child_process';
+
+const fixupProposal = (proposal: ProposalInfo) => {
+  const proposalPath = path.join(
+    'proposals',
+    `${proposal.proposalIdentifier}:${proposal.proposalName}`,
+  );
+  console.log('checking proposal', proposalPath, '...');
+  const packageJson = JSON.parse(
+    fs.readFileSync(path.join(proposalPath, 'package.json'), 'utf-8'),
+  );
+  if (packageJson.dependencies) {
+    assert(
+      packageJson.packageManager,
+      'missing packageManager in package.json',
+    );
```

Comment:
This will stop after reporting the first problem, right? good enough unless/until it becomes painful...

Reply:
Clarified in the console output